lib/import-export.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+const joi = require('joi');
+const fs = require('fs');
+const mime = require('mime');
+const { featureShema } = require('./routes/admin-api/feature-schema');
+const strategySchema = require('./routes/admin-api/strategy-schema');
+const NotFoundError = require('./error/notfound-error');
+const getLogger = require('./logger');
+const yaml = require('js-yaml');
+const {
+    FEATURE_CREATED,
+    FEATURE_UPDATED,
+    FEATURE_REVIVED,
+    STRATEGY_CREATED,
+    STRATEGY_UPDATED,
+} = require('./event-type');
+
+const logger = getLogger('import-export.js');
+
+const dataSchema = joi.object().keys({
+    features: joi
+        .array()
+        .optional()
+        .items(featureShema),
+    strategies: joi
+        .array()
+        .optional()
+        .items(strategySchema),
+});
+
+async function importDataFromFile(config) {
```

Comment:
I think import/export should be implemented as a class with two public functions: `import` and `export`.

Reply:
done